pkg/trace/config/config.go

```diff
@@ -311,8 +311,9 @@ type AgentConfig struct {
 	Endpoints []*Endpoint
 
 	// Concentrator
-	BucketInterval   time.Duration // the size of our pre-aggregation per bucket
-	ExtraAggregators []string
+	BucketInterval              time.Duration // the size of our pre-aggregation per bucket
+	PeerServiceStatsAggregation bool          // enables/disables stats aggregation for peer.service
```

Comment:
```suggestion
	PeerServiceAggregation bool          // enables/disables stats aggregation for peer.service
```

Reply:
I've updated this field and the config field to be consistent with `PeerServiceAggregation`.